src/game_initialization/multiplayer.cpp

```diff
@@ -682,8 +682,8 @@ void start_local_game_commandline(CVideo& video, const config& game_config, save
 	}
 
 	std::string label = "";
```

Comment:
But is that line important? Maybe it was there for a reason? (I don't really know what that does.)